### DISCUSSION TITLE
empty iframe

### DISCUSSION THREAD

**Discussion by abdo-spices** [facebook/docusaurus#8407] [Q&A]
i want to add iframe for a an empty page (,html or .jsx)
but they will not work at build

![image](https://user-images.githubusercontent.com/93441520/205446199-f2205b52-581b-4f1b-8b0e-be40b6555650.png)
![image](https://user-images.githubusercontent.com/93441520/205446213-f10a3416-98a2-4007-a7db-0331b7e6928d.png)
```mdx
<iframe src="pathname:///assets/e" width="100%" height="100%">
</iframe>
```
![image](https://user-images.githubusercontent.com/93441520/205446272-3c2a2731-eb9d-49f0-9afb-6f0410ef8beb.png)

**Comment by slorber** [comment]
pathname is a protocol that we use on Docuaurus as a workaround to disable our link/image URLs logic (like applying baseurl or converting to require calls)

If you want to use an iframe you shouldn't use it.

**Reply by abdo-spices** [reply]
ok what should i do @slorber  @homotechsual said  you have to use  pathname

**Reply by slorber** [reply]
doc: https://docusaurus.io/docs/advanced/routing#escaping-from-spa-redirects
